docs/composefs.md

```diff
@@ -30,21 +30,13 @@ have a `.ostree.cfs` file in the deployment directory which is a mountable
 composefs metadata file, with a "backing store" directory that is
 shared with the current `/ostree/repo/objects`.
 
-### Kernel argument ot-composefs
+### composefs configuration
 
-The `ostree-prepare-root` binary will look for a kernel argument called `ot-composefs`.
+The `ostree-prepare-root` binary will look for `ostree/prepare-root.conf` in `/etc` and
+`/usr/lib` in the initramfs. Using that configuration file you can enable composefs,
+and specify a ed25519 public key to validate the booted commit.
```

Comment:
```suggestion
and specify an ed25519 public key to validate the booted commit.
```

Reply:
Surely it is "a key", not "an key"?

Reply:
I think rabellew is right here, because ed25519 starts with a vowel.  It's about the immediately following word, not the logical subject.  I think basically because it's about smoothing out pronunciation.